.appveyor.yml

```diff
@@ -7,7 +7,7 @@ environment:
   CYG_ROOT: C:\cygwin64
   matrix:
     - OPAM_SWITCH: 4.04.0+msvc32
-    - OPAM_SWITCH: 4.04.0+mingw64c
+    - OPAM_SWITCH: 4.04.0+mingw64
```

Comment:
The mingw boostrap doesn't work out of the box with cygwin. The gcc compiler shipped by cygwin is a a cross compiler (from cygwin to native windows). `$(CC) -MM` (https://github.com/ocaml-omake/omake/blob/d54b75cb237c34caf7689568c7df957713557d78/lib/build/C.om#L407) will therefore output cygwin style paths, that are not understood by omake .
And `4.04.0+mingw64[c]?` wouldn't work anyway. I've only patched `4.04.2+mingw64` with https://caml.inria.fr/mantis/view.php?id=7638 to work around a mingw-w64 bug (threaded bytecode, as used by omake during bootstraping, is prone to random crashes without it - that's probably what you are seeing at AppVeyor).

Reply:
Thanks for looking into this. As far as I remember, the path issue is resolved already, and the build would have stopped much earlier if not. So I generally expect that omake works fine with the mingw toolchains from Cygwin.

I was not aware of the problem with threaded bytecode, thanks for pointing this out. Let me try whether 4.04.2 works.

Reply:
Ok, so finally the build is successful. Turned out that there is in deed a path problem, but I could solve it by setting the path to the ocaml standard library with `-isystem` instead of `-I`. That way, C headers included with the angle bracket notation (like `#include <caml/misc.h>`) are never printed, and we get around the issue.

Reply:
Yes, omake itselfs build. But anyone who tries do compile his own stub code, will stumble upon the same problem. Perhaps document the workaround somewhere.